app-metadata/net.thunderbird.android.beta/en-US/full_description.txt

```diff
@@ -0,0 +1,38 @@
+Help make the next Thunderbird release as awesome as possible by downloading Thunderbird Beta and getting early access to the latest features and bug fixes before they're officially released. Your testing and feedback are important, so please report bugs, rough edges, and share your thoughts with us!
+
+Find our bug tracker, source code, and wiki at <a href="https://github.com/thunderbird/thunderbird-android">https://github.com/thunderbird/thunderbird-android</a>.
+
+We’re always happy to welcome new developers, designers, documenters, translators, bug triagers and friends. Visit us at <a href="https://thunderbird.net/participate">https://thunderbird.net/participate</a> to get started.
+
+What you can do
+Thunderbird is a powerful, privacy-focused email app. Effortlessly manage multiple email accounts from one app, with a Unified Inbox option for maximum productivity. Built on open-source technology and supported by a dedicated team of developers alongside a global community of volunteers, Thunderbird never treats your private data as a product. Supported solely by financial contributions from our users, so you never have to see ads mixed in with your emails again.
+
+<h2>What you can do</h2>
```

Comment:
"What you can do" is present twice. Should this be one heading with a paragraph of text, followed by a list?

Reply:
Yes, one initial set of paragraphs, then multiple lists with a heading. Thank you for catching!